.pre-commit-config.yaml

```diff
@@ -66,6 +66,7 @@ repos:
     - --comment-style
     - '#'
     types: [python]
+    exclude: scripts/data_prep/validate_and_tokenize_data.py
```

Comment:
was there a particular reason this was excluded just curious?

Reply:
For databricks to render a python script as a notebook, it needs the script to start with #databricks notebook source. This change asks pre-commit to skip adding the license header to the script.